docker/spec/fixtures/Puppetfile

```diff
@@ -1,2 +1,2 @@
-moduledir 'test/modules'
+moduledir '/tmp/modules'
```

Comment:
Not sure if this is a concern but, that's gone from a xplat path to a unix only path. Given R10K is xplat, should this be changed?

Reply:
@glennsarti good catch. I don't think this is a concern since this Puppetfile is installed on the container, which will always be linux-y